### PR TITLE
1.13.0

### PR DESCRIPTION
**Pull request by Wizwizard**

### CODE REVIEW

**Comment by vdiravka** [comment]
@Wizwizard Please follow this guideline while opening PR:
https://drill.apache.org/docs/apache-drill-contribution-guidelines/#step-3:-get-your-code-reviewed-and-committed-to-the-project.
* Create Jira with description what you intend to improve (or find it from existing Jira list)
* Translate Chinese into English
* Remove you helper comments and loggers, which are unnecessary for master.
* If possible try to make PR with one commit

File: exec/java-exec/src/main/resources/rest/query/query.ftl [hunk 1]
```diff
@@ -21,11 +21,12 @@
   <script src="/static/js/ace-code-editor/ext-language_tools.js" type="text/javascript" charset="utf-8"></script>
   <script src="/static/js/ace-code-editor/theme-sqlserver.js" type="text/javascript" charset="utf-8"></script>
   <script src="/static/js/ace-code-editor/snippets/sql.js" type="text/javascript" charset="utf-8"></script>
-  <script src="/static/js/ace-code-editor/mode-snippets.js" type="text/javascript" charset="utf-8"></script>
+  <script src="/static/js/ace-code-editor/mode-snippets.js" type="text/javascript" charset="utf-8"></script>
```
Comment (kkhatua) [inline exec/java-exec/src/main/resources/rest/query/query.ftl]: There seems to be an unprintable character you might have accidentally introduced, and should be removed

File: exec/java-exec/src/main/resources/rest/query/query_bak.ftl [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+<#include "*/generic.ftl">
+<#macro page_head>
+    <#if model?? && model>
+      <script src="/static/js/jquery.form.js"></script>
+    </#if>
+    <script type="text/javascript">
+        window.onload = function()
+        {
+              $.get("/data", {username: "bingxing.wang"},function(data, status){
```
Comment (kkhatua) [inline exec/java-exec/src/main/resources/rest/query/query_bak.ftl]: What is the purpose of this file? Please remove any default usernames not required.

File: exec/java-exec/src/main/java/org/apache/drill/exec/client/DrillClient.java [hunk 3]
```diff
@@ -170,6 +170,8 @@ public DrillClient(DrillConfig config, ClusterCoordinator coordinator, BufferAll
     this.reconnectTimes = config.getInt(ExecConstants.BIT_RETRY_TIMES);
     this.reconnectDelay = config.getInt(ExecConstants.BIT_RETRY_DELAY);
     this.supportComplexTypes = config.getBoolean(ExecConstants.CLIENT_SUPPORT_COMPLEX_TYPES);
+
+    logger.debug("picasso: DrillClient(): ownsZkConnection:" + this.ownsZkConnection);
```
Comment (kkhatua) [inline exec/java-exec/src/main/java/org/apache/drill/exec/client/DrillClient.java]: The log message is cryptic and not clear.

File: exec/java-exec/src/main/java/org/apache/drill/exec/client/DrillClient.java [hunk 4]
```diff
@@ -577,6 +581,8 @@ public Version getServerVersion() {
     checkArgument(type == QueryType.LOGICAL || type == QueryType.PHYSICAL || type == QueryType.SQL,
         String.format("Only query types %s, %s and %s are supported in this API",
             QueryType.LOGICAL, QueryType.PHYSICAL, QueryType.SQL));
+    //bingxing.wang.log
```
Comment (kkhatua) [inline exec/java-exec/src/main/java/org/apache/drill/exec/client/DrillClient.java]: Clean up needed.

Comment (Wizwizard) [reply exec/java-exec/src/main/java/org/apache/drill/exec/client/DrillClient.java]: I am sorry to have brought you trouble.    I just want to merge drill 1.13.0 to my project 1.13.0 (what I have modified some code for personal use).  But it seems that I have made a mistake.

Comment (Wizwizard) [reply exec/java-exec/src/main/java/org/apache/drill/exec/client/DrillClient.java]: Please ignore this commit. Very sorry to have brought you trouble.

**Pull request closed without merging**